File: Data Structures & Algorithms/duplicate-integer/submission-0.py
class Solution:
    def hasDuplicate(self, nums: List[int]) -> bool:
        nums.sort()
        cnt = 1
        if len(nums) == 1: return False
        for i in range(1, len(nums)):
            if nums[i-1] == nums[i]:
                cnt += 1
        if cnt >= 2:
            return True
        else: return False

File: Data Structures & Algorithms/palindromic-substrings/submission-0.cpp
class Solution {
public:
    bool t[1000][1000];
    int countSubstrings(string s) {
        int n = s.length();
        int cnt = n;
        for(int i = 0; i < n; i++){
            t[i][i] = true;
        }
        for(int len = 2; len<= n; len++){
            for(int i = 0; i < n - len + 1; i++){
                int j = i + len - 1;
                if(s[i] == s[j] && len == 2){
                    t[i][j] = true;
                    cnt++;
                }
                else if(s[i] == s[j] && t[i+1][j-1] == true){
                    cnt++;
                    t[i][j] = true;
                }
                else{
                    t[i][j] = false;
                }
            }
        }
        return cnt;
    }
};